test/e2e/parallel/spacerequest_test.go

```diff
@@ -279,90 +279,3 @@ func TestCreateSpaceRequest(t *testing.T) {
 		require.NoError(t, err)
 	})
 }
-
-// Update scenarios are not supported as of now since:
-// - the only tier that works with this controller is `appstudio-env`,
-//   the controller tries to create a TokenRequest for the `namespace-manager` SA , and the only tier that has this SA is the `appstudio-env` tier
-// - updating the cluster roles in the SpaceRequest doesn't provision the namespace to a different cluster, as of now this feature is not available.
-//
-//func TestUpdateSpaceRequest(t *testing.T) {
-//	// given
-//	t.Parallel()
-//	// make sure everything is ready before running the actual tests
-//	awaitilities := WaitForDeployments(t)
-//	hostAwait := awaitilities.Host()
-//	memberAwait := awaitilities.Member1()
-//
-//	// when
-//	spaceRequest, parentSpace := CreateSpaceRequest(t, awaitilities, memberAwait.ClusterName,
-//		WithSpecTierName("appstudio"),
-//		WithSpecTargetClusterRoles([]string{cluster.RoleLabel(cluster.Tenant)}))
-//
-//	// then
-//	// check for the subSpace creation
-//	subSpace, err := awaitilities.Host().WaitForSubSpace(t, spaceRequest.Name, spaceRequest.Namespace, parentSpace.GetName(),
-//		UntilSpaceHasAnyTargetClusterSet(),
-//		UntilSpaceHasTier("appstudio"),
-//		UntilSpaceHasAnyProvisionedNamespaces(),
-//	)
-//	require.NoError(t, err)
-//	VerifyResourcesProvisionedForSpace(t, awaitilities, subSpace.Name,
-//		UntilSpaceHasAnyTargetClusterSet(),
-//		UntilSpaceHasTier(spaceRequest.Spec.TierName),
-//	)
-//	spaceRequestNamespacedName := types.NamespacedName{Namespace: spaceRequest.Namespace, Name: spaceRequest.Name}
-//	_, err = memberAwait.WaitForSpaceRequest(t, spaceRequestNamespacedName,
-//		UntilSpaceRequestHasTierName("appstudio"),
-//		UntilSpaceRequestHasConditions(wait.Provisioned()),
-//		UntilSpaceRequestHasNamespaceAccess(subSpace),
-//	)
-//	require.NoError(t, err)
-//	VerifyNamespaceAccessForSpaceRequest(t, memberAwait.Client, spaceRequest)
-//
-//	t.Run("update space request tierName", func(t *testing.T) {
```

Comment:
as mentioned above this subtest could be enabled/implemented

Reply:
Reenabled. Thanks! 🚀